data_scrapers/sonoma.py

```diff
@@ -38,6 +38,22 @@ def get_cells(row: element.ResultSet) -> List[str]:
     """
     return [el.text for el in row.find_all(['th', 'td'])]
 
+def row_list_to_dict(row_list: List[str], headers: List[str]) -> Dict[str, str]:
+    output = {}
+    for i in range(len(row_list)):
+        val = row_list[i]
+        header = headers[i]
+        output[header] = val
+    return output
```

Comment:
This whole function can actually just be done with:

```py
dict(zip(headers, row_list))
```

So you could simplify this function to:

```py
def row_list_to_dict(row_list, headers):
    return dict(zip(headers, row_list))
```

Or get rid of it entirely and replace line 55 with:

```py
return [dict(zip(header_cells, row)) for row in body_cells]
```